src/dswx_sar/initial_threshold.py

```diff
@@ -190,7 +190,7 @@ def select_tile_twele(self,
         mean_int = np.nanmean(intensity_gray)
         sig_int = np.nanstd(intensity_gray)
         cvx = sig_int / mean_int
-        rx = mean_int / mean_int_glob
+        rx = mean_int / mean_intensity_glob
```

Comment:
More like a nit pick than a crucial. This function for example is using `mean_intensity_glob` but I've found `mean_intensity_global` is being used in another function.

Reply:
Sorry. Would you let me know where the `mean_intensity_global` is being used?

Reply:
[here](https://github.com/oberonia78/DSWX-SAR/blob/52d83f09061c5ee5d256d336e081e280ecd149d2/src/dswx_sar/initial_threshold.py#L363)

Reply:
Fixed.